build_tools/littlecheck.py

```diff
@@ -6,7 +6,6 @@
 from __future__ import print_function
 
 import argparse
-from collections import deque
```

Comment:
I think all or most changes to littlecheck are upstream and should go away on a rebase?

Reply:
Yes, I believe so. I upstreamed all my littlecheck changes before using them in this branch. I might have botched the rebase, I'm not sure.

Reply:
Oh, it's `master` is on a particularly old version of `littlecheck.py`. The `ctest` branch is on a version in between that and littlecheck's `master`. I'll just merge in the latest.